backend/accounts/transaction.go

```diff
@@ -152,12 +155,12 @@ func NewOrderedTransactions(txs []*TransactionData) OrderedTransactions {
 		case TxTypeSend:
 			balance.Sub(balance, tx.Amount.BigInt())
 			// Subtract fee as well.
-			if tx.Fee != nil {
+			if tx.Fee != nil && !tx.FeeIsDifferentUnit {
```

Comment:
But when the fee is a different unit, where does it go?

Reply:
If you make an erc20 token tx, the fee is paid from the main Ethereum account. There is a 0-value transaction shown in the ETH account for each erc20 tx, that only has fees.

In the portfolio, this is accounted for in the Ethereum timeseries.

Reply:
In other words, if the fee is paid in a different unit, it does not affect the account's balance or timeseries.

Reply:
Ah ok. But the actual fee is still accounted for when computing ETH balance, right.

>  if the fee is paid in a different unit, it does not affect the account's balance or timeseries.

The `FeeIsDifferentUnit` made me think it should still be substracted but in a different way.
How about naming it a `PhantomFee` or `FeeIsPhantom`, similar to `PhantomData` in Rust :)

Anyway, as you prefer. If you keep the original name, add a comment to `FeeIsDifferentUnit` field that "when true, the fee amount does not count towards the balance of the account associated with the transaction because it's already paid in another transaction"?

Reply:
Oops saw it too late. Here: https://github.com/digitalbitbox/bitbox-wallet-app/pull/1039